README.md

```diff
@@ -41,114 +41,113 @@ cyrusauth:
 
 In the datadir are only two files:
 
 - `znc.pid` - The pid of the currently running ZNC instance.
-- `znc.pem` - This is the server certificate ZNC uses for listening and is created
- with `znc --makepem`.
+- `znc.pem` - This is the server certificate ZNC uses for listening and is
+created with `znc --makepem`.
 
 These directories are also in there:
 
-- configs - Contains `znc.conf` (ZNC's config file) and backups of older configs.
+- configs - Contains `znc.conf` (ZNC's config file) and backups of older
+  configs.
 - modules - ZNC also looks in here for a module.
 - moddata - Global modules save their settings here.
   (e.g. webadmin saves the current skin name in here)
-- users   - This is per-user data and mainly contains just a moddata directory.
+- users   - This is per-user data and mainly contains just a moddata
+  directory.
 
 ## ZNC's config file
 
 This file shouldn't be too hard too understand. An explanation of all the
-items can be found on the [Configuration](http://wiki.znc.in/Configuration)-Page.
-Warning: better not to edit config, while ZNC is running.
+items can be found on the
+[Configuration](http://wiki.znc.in/Configuration)-Page.
+**Warning: better not to edit config, while ZNC is running.**
 
-To rehash the config file, you can send ZNC SIGHUP via
-	pkill -SIGHUP znc
-or you can login to ZNC and use
-	/msg *status rehash
+To rehash the config file, you can send ZNC SIGHUP via `pkill -SIGHUP znc`
+or you can login to ZNC and use `/msg *status rehash`
 
 If you changed some settings while ZNC is running, a simple
-	pkill -SIGUSR1 znc
-will make ZNC rewrite its config file. Alternatively you can use this:
-	/msg *status saveconfig
+`pkill -SIGUSR1 znc` will make ZNC rewrite its config file. Alternatively
+you can use `/msg *status saveconfig`
 
 ## Writing own modules
 
 You can write your own modules in either C++, python or perl.
 
-C++ modules are compiled by either saving them in the modules source dir and
-running make or with the znc-buildmod shell script.
+C++ modules are compiled by either saving them in the modules source dir
+and running make or with the `znc-buildmod` shell script.
 
 For additional info look in the wiki:
 
 - [Writing modules](http://wiki.znc.in/Writing_modules)
 
-Perl modules are loaded through the global module [ModPerl](http://wiki.znc.in/Modperl).
+Perl modules are loaded through the global module
+[ModPerl](http://wiki.znc.in/Modperl).
 
-Python modules are loaded through the global module [ModPython](http://wiki.znc.in/Modpython).
+Python modules are loaded through the global module
+[ModPython](http://wiki.znc.in/Modpython).
 
 ## Further infos
 
-Please visit http://znc.in/ or #znc on freenode if you still have questions.
+Please visit http://znc.in/ or
+[#znc on freenode](ircs://irc.freenode.net:6697/#znc) if you still have
+questions.
 
 You can get the latest development version with git:
-	git clone git://github.com/znc/znc.git
+`git clone https://github.com/znc/znc.git`
```

Comment:
git isn't encrypted and I think GitHub wishes people to start moving to https as they are hiding the git clone links.